src/style.css

```diff
@@ -0,0 +1,37 @@
+:root {
+  --bg-active: url("./images/star-active.svg") center no-repeat;
+  --bg-inactive: url("./images/star.svg") center no-repeat;
+  --size: 16px;
+  --star-distance: 4px;
+}
+
+.body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: var(--size);
+  width: var(--size);
+  background: var(--bg-inactive);
+  margin-right: var(--star-distance);
+}
+
+.inactive {
+  background: var(--bg-inactive);
+}
+
+.active {
+  background: var(--bg-active);
+}
+
+.stars--1 > .stars__star:not(:nth-child(n + 2)),
+.stars--2 > .stars__star:not(:nth-child(n + 3)),
```

Comment:
```suggestion
.stars--2 > .stars__star:nth-child(-n + 2),
```

Reply:
You can make it simpler 🙂

Reply:
Thank you for review! I've made it according to the second remark/ and then I've tried to replace initial active and inactive state of stars. Solution is pretty much the same, but with all positive digits.

Reply:
Well, solution is same, but now it has opposite logic 🙂 But ok